db/post_migrate/20210621234939_backfill_email_log_topic_id.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class BackfillEmailLogTopicId < ActiveRecord::Migration[6.1]
+  disable_ddl_transaction!
+  BATCH_SIZE = 30_000
+
+  def up
+    offset = 0
+    email_log_count = EmailLog.count
```

Comment:
We need to avoid using ActiveRecord in migrations :)

Reply:
Argh every time!!  Will fix